### PR TITLE
Add live code share guide

### PR DESCRIPTION
**Pull request by vitaliiprodan**
Why:

* Explain team members how to use tools for live sharing code to use
  that for purposes like pair programming interview or just plain pair
  programming session

### CODE REVIEW

File: guides/tools/live-code-share.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Live Code Sharing
+
+During the 4 hours pair programming interview you may need to use some tool to share the code from specific workspace. Or another use case is pair programming session with another teammate. There are a lot of different approaches to do that, however this one proposed by DVELP team.
```
Comment (alvaro-chalar) [inline guides/tools/live-code-share.md]: During the 4 hour pair programming interview we may need a tool to share code from a specific workspace or perhaps have a programming session with one of our teammates. From all the different approaches out there, at DVELP we propose the following:

File: guides/tools/live-code-share.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+1. Install [Visual Studio Code](https://code.visualstudio.com), if not installed.
+2. Install [Live Share Extension Pack](https://marketplace.visualstudio.com/items?itemName=MS-vsliveshare.vsliveshare-pack)
+3. Restart Visual Studio Code app
+4. Sign in with your GitHub account. To do that press `command+shift+P` for Mac or `control+shift+P` for Windows/Linux machines and type "Live Share: Sign In With Browser" -> "Sign in with GitHub"
+
+![Live Code Share](../../assets/live-code-share-1.png)
+
+5. Follow "Authorize Visual Studio Live Share" steps in your browser to continue singing in process
+6. After this you should be able to start live session. To do that, click on relevant icon of "Live Share" extension on your activity bar
```
Comment (alvaro-chalar) [inline guides/tools/live-code-share.md]: ...you should be able to start `a` live session. To do that, click on `the` relevant icon...

File: guides/tools/live-code-share.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+![Live Code Share](../../assets/live-code-share-2.png)
+
+7. Now you can start your live session simply by clicking on "Start collaboration session...". It should also copy invitation link to your clipboard automatically, so you can share it with relevant person(s)
```
Comment (alvaro-chalar) [inline guides/tools/live-code-share.md]: Now`,` ....It should also copy `an` invitation link.........so you can share it with `your peer`.

File: guides/tools/live-code-share.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+8. To stop your live session click on stop icon on "Session Details" panel
+
+![Live Code Share](../../assets/live-code-share-3.png)
+
+## Other Links
+
+For more information visit [official web-site](https://visualstudio.microsoft.com/services/live-share) of the extension.
```
Comment (alvaro-chalar) [inline guides/tools/live-code-share.md]: visit `the`

**Review by alvaro-chalar — COMMENTED** [review]
@vitaliiprodan My 2 cents! :wink:

**Comment by vitaliiprodan** [comment]
Thanks @alvaro-chalar. That's fixed 👍

**Review by alvaro-chalar — APPROVED** [review]

**Review by ripixel — APPROVED** [review]

**Pull request merged**